### PR TITLE
Initial site structure setup and first basic informations for topic 'i18n'

### PR DESCRIPTION
**Pull request by marianfoo**
Hallo Volker,
habe mal die ersten Informationen zusammen geschrieben.
Die meiste Zeit hat noch für das Setup und Verstehen von Jekyll gedauert.
Mit Markdown bin ich auch nicht so vertraut, aber es wird langsam.

### CODE REVIEW

File: docs/i18n/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: default
+title: i18n
+permalink: /i18n/
+has_children: true
+nav_order: 1
+---
+
+i18n
```
Comment (vobu) [inline docs/i18n/index.md:9]: probably add some content on that initial page so it doesn't look so "empty"?

File: docs/i18n/overview.markdown [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+## Language Tags
+To identify the individual languages, so-called "Language Tags" are used which are defined by [BCP-47] (http://www.iana.org/assignments/language-subtag-registry/language-subtag-registry).
+These language tags are stored in the SAP system in table T002 for all stored languages.
+These correspond to the ["ISO 639 alpha-2 language code"](https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes). The SAPUI5 application then converts the determined language key into a BCP-47 tag.
+
+> As a best practice, it should be defined that the BCP-47 language code
+> should be kept as short as possible. When creating it, you should
+> avoid regions, script or subtags as far as possible - unless you add
+> important and helpful information.
+> Good example:
```
Comment (vobu) [inline docs/i18n/overview.markdown:23]: this looks as if a list is wanted, so "-" it :)
might also make them md formatted code words via backslashes, à la `i18n.properties`

File: docs/i18n/overview.markdown [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+i18n.properties
+i18n_DE.properties
+Bad example:
+i18n.properties
```
Comment (vobu) [inline docs/i18n/overview.markdown:27]: already in "good example"?

File: docs/i18n/overview.markdown [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+nav_order: 2
+---
+
+# Overview i18n Overview
+Many companies use SAP systems and applications and many different countries.
+To ensure easy translation, the "i18n" (Internationalization) process is used.
+The app is not always recreated, but only the texts are translated and replaced.
+
+## Language Tags
+To identify the individual languages, so-called "Language Tags" are used which are defined by [BCP-47] (http://www.iana.org/assignments/language-subtag-registry/language-subtag-registry).
```
Comment (vobu) [inline docs/i18n/overview.markdown:15]: BCP-47 should be the link text, right? so removing the " " between [...] and (...) fixes that

File: docs/i18n/usage.markdown [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    *Folder Structure and Init File for i18n*
+
+
+## Usage in XML Views
+
+Since we have defined a model in manifest.json, we access the individual keys in the XML view as with any other model.
+
+The i18n model is referenced first in curly brackets and then the key:
+"{i18n>streetNameNumber}"
+
```
Comment (vobu) [inline docs/i18n/usage.markdown:45]: just as a hint: further structuring the i18n entries is possible à la
mainview.heading.title=Title
-> better overview in large i18n projects
might mention this in this guide, but that's all up to you

File: docs/i18n/usage.markdown [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+We take as an example the already previously defined translations
+
+![i18n Simple Example EN File](img/i18n_en_simple.png)
+:--: 
+*i18n Simple Example EN File*
+
+![i18n example XML View](img/i18nXMLViewExample.png)
+:--: 
+*i18n example XML View*
```
Comment (vobu) [inline docs/i18n/usage.markdown:54]: how about adding placeholders for content in i18n texts?
```js
helloMsg=Hello {0}
//...
oBundle.getText("helloMsg", [sRecipient])
```
might be worth mentioning or at least linking to the respective ui5 doc chapter.

**Review by vobu — CHANGES_REQUESTED** [review]
really like this initial effort, top!
in general, i'd prefer mentioning "UI5" instead of "openUI5" or "SAPUI5", but that's all up to you :)

File: docs/i18n/overview.markdown (hunk 3 above)
Comment (marianfoo) [reply docs/i18n/overview.markdown:27]: i didn't know how best to format.
I wanted to use "good/bad" in order not to forget it.
Should I take it out?

File: docs/i18n/usage.markdown (hunk 5 above)
Comment (marianfoo) [reply docs/i18n/usage.markdown:45]: Good remark
I did not know that before. Have mainly used CamelCase for this case (eg. mainviewHeadingTitle).
Is this a matter of taste or best practice?

File: docs/i18n/usage.markdown (hunk 6 above)
Comment (marianfoo) [reply docs/i18n/usage.markdown:54]: Right, I had already in mind but not added yet

File: docs/i18n/overview.markdown (hunk 3 above)
Comment (vobu) [reply docs/i18n/overview.markdown:27]: https://github.github.com/gfm/#tables-extension- ?

File: docs/i18n/overview.markdown (hunk 4 above)
Comment (marianfoo) [reply docs/i18n/overview.markdown:15]: changed

File: docs/i18n/usage.markdown (hunk 6 above)
Comment (vobu) [reply docs/i18n/usage.markdown:54]: why not mention both and add an example for each?

File: docs/i18n/overview.markdown (hunk 2 above)
Comment (marianfoo) [reply docs/i18n/overview.markdown:23]: ok, i tried to improve it

File: docs/i18n/index.md (hunk 1 above)
Comment (marianfoo) [reply docs/i18n/index.md:9]: should I better take "Overview" as nav_order:1 for now?

Comment (vobu) [reply docs/i18n/index.md:9]: sure - but then the chapter has only one sub-page.
it's considered not-so-good practice to have a chapter (list) w/ only one page (list item).
so try to think of a layout that either has
- 2 subpages with an overview text/page (yeah!)
- single page with all content (meh)

**Review by marianfoo — COMMENTED** [review]
You are right, I personally am of course SAPUI5 centered, we would still have to consider whether we differentiate there to OpenUI5, there are some differences.

File: docs/i18n/index.md (hunk 1 above)
Comment (marianfoo) [reply docs/i18n/index.md:9]: it´s just the beginning, so i plannend more subpages, that´s why i created this subpages

File: docs/i18n/overview.markdown (hunk 3 above)
Comment (marianfoo) [reply docs/i18n/overview.markdown:27]: ?

image

File: docs/i18n/usage.markdown (hunk 6 above)
Comment (marianfoo) [reply docs/i18n/usage.markdown:54]: will do on the next request

File: docs/i18n/index.md (hunk 1 above)
Comment (vobu) [reply docs/i18n/index.md:9]: 👍

**Review by vobu — APPROVED** [review]

**Comment by marianfoo** [comment]
i did not commit the suggested changes yet, shall i create a new request?

**Comment by vobu** [comment]
yes, please do that - and keep the PR in status "draft" until finished, then reviewers know when to...well...review :)

**Pull request merged**